### PR TITLE
Update "report a bug" link in footer

### LINKED ISSUE #16884: report a bug link at bottom of extension edit page goes to 404 error
### Describe the problem and steps to reproduce it:

when i visit the following page
https://addons.mozilla.org/en-US/developers/addon/revert-site/edit
there is a "Report a bug" link at the bottom of the page
when i click it, it goes to a 404 error page

### What happened?
clicking the "Report a bug" link goes to a 404 error at  https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/AMO/Policy/Contact 

### What did you expect to happen?
it should take me to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/Contact_us

### PR DESCRIPTION
**Pull request by bhushan-borole**
Fixes #16884 

@willdurand

### CODE REVIEW

File: src/olympia/templates/photon-footer.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h4 class="Footer-links-header"><a href="{{ url('home') }}">Add-ons</a></h4>
                 <li><a href="{{ url('devhub.index') }}">Developer Hub</a></li>
                 <li><a href="https://developer.mozilla.org/docs/Mozilla/Add-ons/AMO/Policy">Developer Policies</a></li>
                 <li><a href="https://discourse.mozilla-community.org/c/add-ons">Forum</a></li>
-                <li><a class="Footer-bug-report-link" href="https://developer.mozilla.org/docs/Mozilla/Add-ons/Contact_us">Report a bug</a></li>
+                <li><a class="Footer-bug-report-link" href=" https://addons.mozilla.org/about">Report a bug</a></li>
```
Comment (willdurand) [inline src/olympia/templates/photon-footer.html:17]:
```suggestion
                <li><a class="Footer-bug-report-link" href="https://addons.mozilla.org/about">Report a bug</a></li>
```

**Review by willdurand — CHANGES_REQUESTED** [review]
There is an extra space that should be removed.

File: src/olympia/templates/photon-footer.html (hunk 1 above)
Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: I am sorry xDDD
Will fix it right away

Comment (willdurand) [reply src/olympia/templates/photon-footer.html:17]: You can probably apply the suggestion I made.

Comment (bhushan-borole) [reply src/olympia/templates/photon-footer.html:17]: Oh I didn't know about this feature

**Review by willdurand — APPROVED** [review]
Looks good to me, thanks

**Comment by bhushan-borole** [comment]
What is this error but, why is it failing?

**Comment by willdurand** [comment]
> What is this error but, why is it failing?

I dunno, it is an unrelated test case. I wouldn't worry about it.

**Pull request merged**